package.json

```diff
@@ -49,6 +49,7 @@
         "@types/react": "^16.14.3",
         "@typescript-eslint/eslint-plugin": "~4.29.0",
         "@typescript-eslint/parser": "~4.29.0",
+        "@webpack-cli/serve": "^1.5.2",
```

Comment:
can remove?

Reply:
I needed it to run `npm start`, is it just assumed people already have this installed?

Reply:
I have not needed it yet, so not sure.  I'm surprised there wouldn't also be a package-lock update then too.